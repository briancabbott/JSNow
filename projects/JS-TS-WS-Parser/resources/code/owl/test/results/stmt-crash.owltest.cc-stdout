input (0 - 11)
  stmt : IF (0 - 11)
    expr (3 - 4)
      identifier - a (3 - 4)
    stmt : EXPR (7 - 9)
      expr (7 - 8)
        identifier - a (7 - 8)
