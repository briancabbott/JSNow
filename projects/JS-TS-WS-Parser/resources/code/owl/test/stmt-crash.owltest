if a { a. }
---
input = stmt*
stmts =
    [ '{' stmt* '}' ] : group
    [ '{' stmt* '}' ] : stmt
stmt =
    'if' expr [ '{' stmt* '}' ] : if
    expr '.' : expr
expr = identifier+
