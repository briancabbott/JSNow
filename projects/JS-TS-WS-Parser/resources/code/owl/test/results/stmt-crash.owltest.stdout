. if a    { a   .     }
  |  |      expr       
  |  expr   stmt:expr  
  stmt:if--------------
